app/common/tables.html

```diff
@@ -0,0 +1,40 @@
+<div class="container margin">
+   <div class="minus"> </div>
+   <div class="minus"></div>
+   <div class="minus"></div>
+   <div class="textstyle">All Outlets</div>
+</div>
+<div class="container">
+   <div class="col-md-12">
+      <div style="margin-top: 40px;">
+         <div class="color">{{tabledetails.tabletitle}}</div>
+         <table class="table">
+            <thead>
+               <tr>
+                  <th class="headingcolor" ng-repeat="data in tabledetails.values">{{data}}</th>
+                  <th></th>
+                  <th  ng-if="data.Type=='SUBMIT'" ng-repeat="data in tabledetails.button">
+                     <button type="submit" class="btn btn-primary"> {{data.Type}}
+                  </th>
+               </tr>
+            </thead>
+            <tbody>
+
+               <tr ng-if="tabledetails.controller=='outletCtrl'" ng-repeat="viewdata in tablebody">
+                  <td>  {{tablebody[$index].value.id}}</td>
```

Comment:
this should be a generic directive. right now this directive will only work for outlet.

you need to make one single directive for all the tables in the entire app